code-challenges.test.js

```diff
@@ -35,35 +51,410 @@ const num2 = 0
 const num3 = -7
 // Expected output: "-7 is not divisible by three"
 
+// First test failed
+
 
 // b) Create the function that makes the test pass.
+// pseudo code: create a function called divByThree
+// using the modulo function create test if number is divisible by 3
+// if value is true, return the number with string "is divisible by three"
+// if value returned is false, return the number with the string " is not divisible by three"
 
+const divByThree = () => {
+    if(divByThree % 3 === 0) {
+        return `${divByThree()} is divisible by three`
+  }  else {
+        return `${divByThree()} is not divisible by three}`
+    }
+}
+console.log(divByThree(num1))
+console.log(divByThree(num2))
+console.log(divByThree(num3))
 
 
+// After creating the function the yarn jest still returned as failed and was stuck on this.
```

Comment:
You are very close. Go back and look in the syllabus to check the structure. The expect statement gives an hint on what your function should do.